automobile/sumo-interface.md

```diff
@@ -4,79 +4,131 @@ An interface with a microscopic traffic simulator called `SUMO` (Simulation of
 
-> **Note** [macOS]:
-On macOS, SUMO relies on X11. You need therefore to install [XQuartz](https://www.xquartz.org) (version 2.7.8 or later) for the interface to work.
+Here is the parameters of the `SumoInterface` PROTO (which inherits from the `Supervisor` node):
+
+```
+PROTO SumoInterface [
+  field SFBool    gui                   TRUE
+  field SFBool    useNetconvert         TRUE
+  field SFBool    enableTrafficLights   TRUE
+  field SFBool    enableWheelsRotation  FALSE
+  field SFInt32   maxVehicules          100
+  field SFInt32   radius               -1
+  field SFFloat   maximumLateralSpeed   2.5
+  field SFFloat   maximumAngularSpeed   3
+  field SFFloat   laneChangeDelay       3
+  field SFBool    enableHeight          FALSE
+  field SFInt32   step                  200
+  field SFInt32   port                  8873
+  field SFInt32   seed                  1
+  field SFString  networkFiles          ""
+  field SFString  sumoArguments         ""
+  field SFNode    display               NULL
+  field SFBool    verbose               FALSE
+  field SFBool    synchronization       TRUE
+  field MFNode    children              []
+]
+```
 
+#### SumoInterface fields summary
+
+- `gui`: Defines if the command-line or GUI version of SUMO should be used.
+- `useNetconvert`: Defines if NETCONVERT should be called before launching SUMO, this should be disabled if a `*.net` is already provided.
+- `enableTrafficLights`: Defines if the traffic lights synchronization should be used or not.
+- `enableWheelsRotation`: Defines if the wheels of the vehicles should rotate or not (wheels rotation has an impact on the simulation speed but make the simulation looks more realistic).
+- `maxVehicules`: Defines the maximum number of vehicles added in Webots from SUMO.
+- `radius`: Defines the visibility radius of the vehicles in meters (only vehicles closer to the viewpoint than this radius will be added in Webots). A negative value means that all the vehicles are added.
+- `maximumLateralSpeed`: Defines the maximal lateral speed of any vehicle in meter per second.
+- `maximumAngularSpeed`: Defines the maximal angular speed of any vehicle in radian per second.
+- `laneChangeDelay`: Defines the time spent to change lane (during this period, the position of the vehicle in Webots and SUMO may not be synchronized anymore).
+- `enableHeight`: Defines whether height information should be extracted from the edge name (experimental).
+- `step`: Defines the time step of SUMO in milliseconds (should be bigger or equal to the world time step).
+- `port`: Defines which port SUMO and Webots should use to communicate.
+- `seed`: Defines the seed of the SUMO random number generator. Use `0` for a random seed (`--random` option of SUMO).
+- `networkFiles`: Defines the directory where are located the SUMO network files. If this field is empty, the files should be located in the same directory than the world file in a directory called `worldName_net`.
```

Comment:
&rarr; Defines the directory where the SUMO network files are stored.
&arra; If this field is empty, the files should be stored in a directory called `worldName_net` located in the same directory as the world file.